pkg/experiment/symbolizer/types.go

```diff
@@ -0,0 +1,41 @@
+package symbolizer
+
+import (
+	googlev1 "github.com/grafana/pyroscope/api/gen/proto/go/google/v1"
+	"github.com/grafana/pyroscope/lidia"
+
+	pprof "github.com/google/pprof/profile"
+)
+
+type locToSymbolize struct {
+	idx int
+	loc *googlev1.Location
+}
+
+// LidiaTableCacheEntry represents a cached Lidia table with its binary layout information
+type LidiaTableCacheEntry struct {
+	Data []byte // Processed Lidia table data
+}
+
+// Location represents a memory address to be symbolized
+type Location struct {
+	ID      string
+	Address uint64
+	Lines   []lidia.SourceInfoFrame
+	Mapping *pprof.Mapping
+}
+
+// Request represents a symbolization request for multiple addresses
+type Request struct {
```

Comment:
I think this is not used outside the package, we can make it not exporter

Reply:
Yes, I also unexported other types not used outside the packages :)